run-backend.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/ash
+
+export GDAL_LIBRARY_PATH='/usr/lib/libgdal.so.34'
+export GEOS_LIBRARY_PATH='/usr/lib/libgeos_c.so.1'
+
+#git clone https://github.com/codewithaloha/froide.git
+#yarn link
+yarn install
+yarn build
+yarn serve dev &
+
+# hack until we get the froide setup build working (i.e. include templates)
+echo "STARTING DB"
+# To initialise the database:
+/root/.local/bin/poetry run ./manage.py migrate --skip-checks
+# Create a superuser
+/root/.local/bin/poetry run ./manage.py createsuperuser
```

Comment:
Creating an admin user (createsuperuser) fails due to:
```
Superuser creation skipped due to not running in a TTY.
```
Additionally, there are migrations that need to be created and applied and seed data should be loaded.

Reply:
TBH, the superuser creation and loading the seed data should probably be gated behind an env variable or setting.  If we're using the container for more than toy demos, we probably don't want this to happen everytime that container is started up.

If migrations are missing, we should create the necessary migration files so that they can be applied automatically.  My understanding is that the migrate command should apply all the migrations that django is aware of so if that skips migrations, we should fix that rather than trying to manually apply migrations.

Reply:
You should use the data/seed/init_db.sh. It will run the migrations, load the seed data, and populate the search indexes. Please see the [Quick Start](https://github.com/CodeWithAloha/uipa/blob/main/docs/Getting-Started.md).

Reply:
I looked at the `init_db.sh` and the only difference is that the  script loads the data at the end.  The rest (migrate and populate) occur in the container as well.  So if migrations are missing, there's something wrong elsewhere.  Also, TBH, we should remote a lot of the initialization stuff from the container startup and gate that since we don't want to reset the environment whenever the container starts.  However, we can address that in another PR.

Reply:
I agreed that there's no need to initialize the database every time the app container is started. The initialization script (whatever that might be) could touch a file (let's call it DB_INIT_DONE or something). When the app container is started after being stopped (not newly created), the initialization script can check for the DB_INIT_DONE file and skip initialization.